tests/unit/test_model_card.py

```diff
@@ -1046,7 +1090,10 @@ def test_training_details_autodiscovery_from_model_overview_autopilot(
         model_overview=model_overview_example, sagemaker_session=session
     )
 
+    # There are 0 required keys in training_metrics field in SEARCH_TRAINING_JOB_AUTOPILOT_EXAMPLE has
```

Comment:
It looks confusing. If you want, you could just add why `SEARCH_TRAINING_JOB_AUTOPILOT_EXAMPLE` result in this issue: `MetricDefinitions` is empty.